packages/flutter/lib/src/painting/box_border.dart

```diff
@@ -10,16 +10,27 @@ import 'borders.dart';
 import 'edge_insets.dart';
 
 /// The shape to use when rendering a [Border] or [BoxDecoration].
+///
+/// Consider using [ShapeBorder] subclasses directly (with [ShapeDecoration]),
+/// instead of using [BoxShape], if the shapes will need to be interpolated.
```

Comment:
maybe use "animated" instead of "interpolated"?  Or maybe both: "interpolated in an animation".  Would it be worth describing here why BoxDecoration or Border doesn't work for animation?

Reply:
clarified.